fv3core/stencils/fv_dynamics.py

```diff
@@ -169,10 +169,19 @@ def do_dyn(state, comm):
         domain=grid.domain_shape_standard(),
     )
     print("DynCore", grid.rank)
+    if timer is not None:
+        # start dyn timer
+        timer.start("Dynamics")
```

Comment:
This label could probably be `dyn_core`? "dynamics" also includes tracer advection and remapping.